pkg/scheduler/apis/config/validation/validation_pluginargs.go

```diff
@@ -75,6 +75,10 @@ func validateNoConflict(presentLabels []string, absentLabels []string) error {
 // with an additional check for .labelSelector to be nil.
 func ValidatePodTopologySpreadArgs(args *config.PodTopologySpreadArgs) error {
 	var allErrs field.ErrorList
+	if args.DisableDefaultConstraints && len(args.DefaultConstraints) > 0 {
+		p := field.NewPath("disableDefaultConstraints")
+		allErrs = append(allErrs, field.Invalid(p, args.DisableDefaultConstraints, "cannot be true when .defaultConstraints are not empty"))
```

Comment:
```suggestion
		allErrs = append(allErrs, field.Invalid(p, args.DisableDefaultConstraints, "is true, but .defaultConstraints are not empty"))
```

(obviously s/true/false/ after switch polarity :) )